frontend/src/app/modules/work-space/dashboard/dashboard.component.html

```diff
@@ -1 +1,98 @@
-<p>dashboard</p>
+<div *ngIf="userProjects; then existActiveProjects else emptyActiveProjects"></div>
```

Comment:
You probably wanted to check array length

Reply:
I changed condition to 'userProjects?.length'